modules/imgproc/src/colormap.cpp

```diff
@@ -727,7 +757,8 @@ namespace colormap
             colormap == COLORMAP_TWILIGHT ? (colormap::ColorMap*)(new colormap::Twilight) :
             colormap == COLORMAP_TWILIGHT_SHIFTED ? (colormap::ColorMap*)(new colormap::TwilightShifted) :
             colormap == COLORMAP_VIRIDIS ? (colormap::ColorMap*)(new colormap::Viridis) :
-            colormap == COLORMAP_WINTER ? (colormap::ColorMap*)(new colormap::Winter) : 0;
+            colormap == COLORMAP_WINTER ? (colormap::ColorMap*)(new colormap::Winter) :
+            colormap == COLORMAP_TURBO ? (colormap::ColorMap*)(new colormap::Turbo) : 0;
```

Comment:
Maybe keep it sorted alphabetically for consistency.

Reply:
@catree Thank you for your comment! I fixed the order of check.